test/DefaultCluster.Tests/ClientAddressableTests.cs

```diff
@@ -7,8 +7,9 @@
 using UnitTests.GrainInterfaces;
 using Xunit;
 
-namespace UnitTests
+namespace DefaultCluster.Tests
 {
+    [TestCategory("BVT")]
```

Comment:
Same as before, please don't add this at class level. And was this added by design? because they weren't BVTs before.